drivers/SmartThings/zigbee-smoke-detector/src/aqara-gas/init.lua

```diff
@@ -0,0 +1,178 @@
+local data_types = require "st.zigbee.data_types"
+local cluster_base = require "st.zigbee.cluster_base"
+local capabilities = require "st.capabilities"
+
+local sensitivityAdjustment = capabilities["stse.sensitivityAdjustment"]
+local sensitivityAdjustmentCommandName = "setSensitivityAdjustment"
+local selfCheck = capabilities["stse.selfCheck"]
+local setSelfCheckStateCommandName = "setSelfCheckState"
+local lifeTimeReport = capabilities["stse.lifeTimeReport"]
+
+local PRIVATE_CLUSTER_ID = 0xFCC0
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+local MFG_CODE = 0x115F
+local PRIVATE_SENSITIVITY_ADJUSTMENT_ATTRIBUTE_ID = 0x010C
+local PRIVATE_MUTE_ATTRIBUTE_ID = 0x0126
+local PRIVATE_SELF_CHECK_ATTRIBUTE_ID = 0x0127
+local PRIVATE_LIFE_TIME_ATTRIBUTE_ID = 0x0128
+local PRIVATE_GAS_ZONE_STATUS_ATTRIBUTE_ID = 0x013A
+
+
+local FINGERPRINTS = {
+    { mfr = "LUMI", model = "lumi.sensor_gas.acn02" }
+}
+
+
+local CONFIGURATIONS = {
+  {
+    cluster = PRIVATE_CLUSTER_ID,
+    attribute = PRIVATE_GAS_ZONE_STATUS_ATTRIBUTE_ID,
+    minimum_interval = 1,
+    maximum_interval = 3600,
+    data_type = data_types.Uint16,
+    reportable_change = 1
+  }
+}
+
+local function gas_zone_status_handler(driver, device, value, zb_rx)
+  if value.value == 1 then
+    device:emit_event(capabilities.gasDetector.gas.detected())
+  elseif value.value == 0 then
+    device:emit_event(capabilities.gasDetector.gas.clear())
+  end
+end
+
+local function buzzer_status_handler(driver, device, value, zb_rx)
+  if value.value == 1 then
+    device:emit_event(capabilities.audioMute.mute.muted())
+  elseif value.value == 0 then
+    device:emit_event(capabilities.audioMute.mute.unmuted())
+  end
+end
+
+local function lifetime_status_handler(driver, device, value, zb_rx)
+  if value.value == 1 then
+    device:emit_event(lifeTimeReport.lifeTimeState.endOfLife())
+  elseif value.value == 0 then
+    device:emit_event(lifeTimeReport.lifeTimeState.normal())
+  end
+end
+
+local function selfcheck_status_handler(driver, device, value, zb_rx)
+  if value.value == 0 then
+    device:emit_event(selfCheck.selfCheckState.idle())
+  elseif value.value == 1 then
+    device:emit_event(selfCheck.selfCheckState.selfCheckCompleted())
+  end
+end
+
+local function sensitivity_adjustment_handler(driver, device, value, zb_rx)
+  if value.value == 0x01 then
+    device:emit_event(sensitivityAdjustment.sensitivityAdjustment.Low())
+  elseif value.value == 0x02 then
+    device:emit_event(sensitivityAdjustment.sensitivityAdjustment.High())
+  end
+end
+
+local function mute_handler(driver, device, cmd)
+  device:send(cluster_base.write_manufacturer_specific_attribute(device,
+    PRIVATE_CLUSTER_ID, PRIVATE_MUTE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 1))
+end
+
+local function unmute_handler(driver, device, cmd)
+  device:send(cluster_base.write_manufacturer_specific_attribute(device,
+    PRIVATE_CLUSTER_ID, PRIVATE_MUTE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 0))
+end
```

Comment:
Hi, @Hongming6 

In the TCs, Aqara mark unmute test case that this device does not support this feature.
If so, the unmute_handler is uncessary.

Reply:
@Hongming6 

In fact the gas detector never process `unmute` command, am I right ?
If so, I'll try to modify the device config to let the `audioMute` only support `mute` command.

Reply:
Can you capture this behavior in a test so that it is clear that it is intended?

i.e. a test of the `unmute` command immediately reporting a `muted` event?